tutorials/workflow/csharp/resiliency-and-compensation/ResiliencyAndCompensation/Program.cs

```diff
@@ -0,0 +1,25 @@
+using Dapr.Workflow;
+using ResiliencyAndCompensation;
+using ResiliencyAndCompensation.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDaprWorkflow(options => {
+    options.RegisterWorkflow<ResiliencyAndCompensationWorkflow>();
+    options.RegisterActivity<MinusOne>();
+    options.RegisterActivity<Division>();
+    options.RegisterActivity<PlusOne>();
+});
+var app = builder.Build();
+
+app.MapPost("/start/{input}", async (int input) => {
+    await using var scope  = app.Services.CreateAsyncScope();
+    var workflowClient = scope.ServiceProvider.GetRequiredService<DaprWorkflowClient>();
```

Comment:
Nope, as in a previous message- this isn't necessary. Inject `DaprWorkflowClient` into the async method (can use an optional `[FromServices]` attribute if you want) and access the instance from there. A separate scope is created by invocation of the inbound post request itself and isn't needed here.